migrations/20220516100946_orders.js

```diff
@@ -10,6 +10,7 @@ exports.up = async (knex) => {
     def.uuid('id').defaultTo(uuidGenerateV4())
     def.string('supplier', 255).notNullable()
     def.specificType('items', 'uuid Array').notNullable()
+    def.string('purchaser_address', '255').notNullable()
```

Comment:
I put 255, just as a max. Could you please update this to  (32 hex digits + 4 dashes) = 36? @mattdean-digicatapult?

Reply:
It's not a uuid it's and address (e.g. `5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty`) so 48 characters.